src/portal-components/DetailsPage/DetailsPageTabs.tsx

```diff
@@ -46,10 +44,7 @@ const DetailsPageTabs: React.FunctionComponent<DetailsPageTabsProps> = (
               to={`${urlWithTrailingSlash}${tab.uriValue + search}`}
               key={`detailPage-tab-${index}`}
               className={'tab-item ignoreLink'}
-              aria-selected={selectedTabIndex === index}
-              onClick={() => {
-                setSelectedTabIndex(index)
-              }}
+              aria-current="true"
```

Comment:
Applies aria-current="true" iff the NavLink is active (i.e. you are on a matching URL)

See https://v5.reactrouter.com/web/api/NavLink/aria-current-string